lib/paper_trail/version_number.rb

```diff
@@ -7,8 +7,8 @@ module PaperTrail
   # because of this confusion, but it's not worth the breaking change.
   # People are encouraged to use `PaperTrail.gem_version` instead.
   module VERSION
-    MAJOR = 12
-    MINOR = 1
+    MAJOR = 13
+    MINOR = 0
```

Comment:
Please don't change version number. First, I like to change it during release process. Second, the next version may be 12.2, as there are no breaking changes yet, AFAIK.

Reply:
Oh, that reminds me, can you please add an entry to the changelog under Unreleased -> Added?

Reply:
Done!